.env.mainnet

```
@@ -0,0 +1,26 @@
+EXPLORER_DAPI_ADDRESSES=https://104.200.24.196:9999,https://134.255.182.185:9999,https://134.255.182.186:9999,https://134.255.182.187:9999,https://134.255.183.247:9999,https://134.255.183.248:9999,https://134.255.183.250:9999,https://135.181.110.216:9999,https://146.59.4.9:9999,https://147.135.199.138:9999,https://149.28.241.190:9999,https://149.28.247.165:9999,https://157.10.199.125:9999,https://157.10.199.77:9999,https://157.10.199.79:9999,https://157.10.199.82:9999,https://157.66.81.130:9999,https://157.66.81.162:9999,https://157.66.81.218:9999,https://157.90.238.161:9999,https://159.69.204.162:9999,https://167.179.90.255:9999,https://167.88.169.16:9999,https://168.119.102.10:9999,https://172.104.90.249:9999,https://173.212.239.124:9999,https://173.249.53.139:9999,https://178.157.91.184:9999,https://185.158.107.124:9999,https://185.192.96.70:9999,https://185.194.216.84:9999,https://185.197.250.227:9999,https://185.198.234.17:9999,https://185.215.166.126:9999,https://188.208.196.183:9999,https://188.245.90.255:9999,https://192.248.178.237:9999,https://193.203.15.209:9999,https://194.146.13.7:9999,https://194.195.87.34:9999,https://198.7.115.43:9999,https://207.244.247.40:9999,https://213.199.34.248:9999,https://213.199.34.250:9999,https://213.199.34.251:9999,https://213.199.35.15:9999,https://213.199.35.18:9999,https://213.199.35.6:9999,https://213.199.44.112:9999,https://2.58.82.231:9999,https://31.220.84.93:9999,https://31.220.85.180:9999,https://31.220.88.116:9999,https://37.27.83.17:9999,https://37.60.236.151:9999,https://37.60.236.161:9999,https://37.60.236.201:9999,https://37.60.236.212:9999,https://37.60.236.247:9999,https://37.60.236.249:9999,https://37.60.243.119:9999,https://37.60.243.59:9999,https://37.60.244.220:9999,https://44.240.99.214:9999,https://49.12.102.105:9999,https://49.13.154.121:9999,https://49.13.193.251:9999,https://49.13.237.193:9999,https://49.13.28.255:9999,https://51.195.118.43:9999,https://51.83.191.208:9999,https://5.189.186.78:9999,https://52.10.213.198:9999,https://52.33.9.172:9999,https://54.69.95.118:9999,https://5.75.133.148:9999,https://64.23.134.67:9999,https://65.108.246.145:9999,https://65.109.65.126:9999,https://65.21.145.147:9999,https://79.137.71.84:9999,https://81.17.101.141:9999,https://91.107.204.136:9999,https://91.107.226.241:9999,https://93.190.140.101:9999,https://93.190.140.111:9999,https://93.190.140.112:9999,https://93.190.140.114:9999,https://93.190.140.162:9999,https://95.216.146.18:9999
+
+# The hostname or IP address of Dash Core RPC interface.
+EXPLORER_CORE_HOST=127.0.0.1
+
+# The port number on which Dash Core RPC is listening.
+EXPLORER_CORE_RPC_PORT=9998
+
+# The username for authentication with Dash Core RPC
+EXPLORER_CORE_RPC_USER=user
+
+# The password for authentication with Dash Core RPC
+EXPLORER_CORE_RPC_PASSWORD=password
+
+# Testnet Insight API URL
+EXPLORER_INSIGHT_API_URL=https://insight.testnet.networks.dash.org:3002/insight-api
```

Should be a mainnet explorer, right?
```suggestion
EXPLORER_INSIGHT_API_URL=https://insight.dash.org/insight/
```